### PR TITLE
modify plugin for copy archive to target folder

### PR DESCRIPTION
**Pull request by leiiiooo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,8 +70,8 @@ cargo install --force cargo-ndk
 
 And now cargo project will be built with your android project:
 
-1. `./gradlew buildDebug` will build `dev` (`debug`) profile
-1. `./gradlew buildRelease` will build `release` profile
+1. `./gradlew buildCargoNdkDebug` will build `dev` (`debug`) profile
```
[the remark this answers is not in the document]
Comment (leiiiooo) [reply README.md:73]: **buildCargoNdkDebug** for this, I mean I just want to compile and copy to target dir. Maybe I donot want to trigger a build process

File: plugin/src/main/groovy/com/github/willir/rust/CargoNdkConfig.groovy [hunk 2]
```diff
@@ -116,6 +116,10 @@ class CargoNdkConfig {
         }
     }
 
+    Path getProjectSrcMainRootPat() {
```
[the remark this answers is not in the document]
Comment (leiiiooo) [reply plugin/src/main/groovy/com/github/willir/rust/CargoNdkConfig.groovy:119]: why we need a path like this s/Pat/Path?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (willir) [reply README.md:73]: Yeah, but I think that the most common case is to build the whole project. I wouldn't just replace `buildDebug` with `buildCargoNdkDebug`.

Maybe it makes sense to say that if you want to build just rust you can use `buildCargoNdkDebug`, however, `buildDebug` would automatically run `buildCargoNdkDebug`?

File: plugin/src/main/groovy/com/github/willir/rust/CargoNdkConfig.groovy (hunk 2 above)
[the remark this answers is not in the document]
Comment (willir) [reply plugin/src/main/groovy/com/github/willir/rust/CargoNdkConfig.groovy:119]: `s/Pat/Path` means replace `Pat` with `Path`.

**Pull request closed without merging**